Src/Test.UnitTests.Sarif.PatternMatcher/AnalyzeCommandTests.cs

```diff
@@ -23,6 +26,24 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher
 {
     public class AnalyzeCommandTests
     {
+        private static AnalyzeOptions CreateDefaultAnalyzeOptions()
```

Comment:
>CreateDefaultAnalyzeOptions [](http://example.com/codeflow?start=38&length=27)

This is a useful helper that uses reflection against the product code to create an AnalyzeOptions instance that is populated with all product default values.

This will prevent errors due to dueling initialization code paths. This code isn't used in this test file, currently, but I intend to come back here over the next week and do other clean-up, so I'm adding it. A lot of code here can be simplified and improved.